src/main/scala/com/microsoft/partnercatalyst/fortis/spark/dto/FortisItem.scala

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.partnercatalyst.fortis.spark.dto
+
+trait FortisItem {
```

Comment:
The point of this is to allow us to easily drop the original item (T) member once analysis is done are we're ready to pass the analysis into the downstream code path. I would have used a case class here, but IIRC, you cannot extend a case class with another case class.